src/components/MatchProfilesForm/MatchProfilesForm.css

```diff
@@ -0,0 +1,10 @@
+@import "@folio/stripes-components/lib/variables.css";
```

Comment:
I assume you do not need this file.

Reply:
@VictorSoroka1 Do you mean MatchProfilesForm.css at all or importing of "@folio/stripes-components/lib/variables.css"?

Reply:
Whole file. It is not used anywhere in your code)

Reply:
@VictorSoroka1 Oh, you are right. I deleted file. Thanks for the comment